_data/reviews.yml

```diff
@@ -381,6 +410,13 @@
       date: 20250916
       name: Thuyet Nguyen
       type: Ad-Hoc
+    - score: 5
+      feedback: |
+        Not yet but it was useful conversatoin.
```

Comment:
I took this verbatim but we can shorten it to `It was useful conversation.` perhaps?